### DISCUSSION TITLE
Soft Deletes (files)

### DISCUSSION THREAD

**Discussion by jpetryk** [directus/directus#5789] [Feature Requests]
In a multiple deployed instance scenario for specific implementations of multi-tenancy (could have share base configurations, default file sets, etc.), "soft deletes" of files make a lot of sense.

This could take the form of a config variable "STORAGE_SOFT_DELETE" (defaults to false), where if enabled, when a user deletes a file it only deletes it's reference to the file within Directus. It remains on the file system/ service in case other instances refer to it.

It might make sense to have additional settings per service as well or instead:

STORAGE_AWS_SOFT_DELETE
STORAGE_LOCAL_SOFT_DELETE

If this is set to true, it just doesn't hard delete it from the underlying service.

**Comment by azrikahar** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
